core/network/build.gradle.kts

```diff
@@ -27,13 +28,24 @@ android {
             )
         }
     }
+
     compileOptions {
         sourceCompatibility = JavaVersion.VERSION_17
         targetCompatibility = JavaVersion.VERSION_17
     }
+
     kotlinOptions {
         jvmTarget = "17"
     }
+
+    buildFeatures {
+        buildConfig = true
+    }
+
+    secrets {
```

Comment:
this should be out `android {...}` block  something like this 

```
secrets {
    defaultPropertiesFileName = "secrets.defaults.properties"
}
```

Reply:
Does the secret block contain only one default file should I remove optional secret.properties file declaration sir